client/src/components/Details/index.js

```diff
@@ -26,7 +27,7 @@ function PaintingsDetail({ match }) {
   };
   useEffect(() => {
     getPaintingByID(match.params.artId);
-  }, []);
+  });
```

Comment:
Removing the dependency array causes an infinite loop, and it makes low-end computers like Rana's get stuck and lagged.